packages/itwin/tree-widget/src/test/trees/category-tree/CategoriesVisibilityHandler.test.ts

```diff
@@ -124,18 +132,18 @@ describe("CategoryVisibilityHandler", () => {
     it("calls enableSubcategoryCategory", async () => {
       await using(createHandler({ activeView: mockViewport().object, categories }), async (handler) => {
         const enableSubCategorySpy = sinon.stub(handler, "enableSubCategory");
-        await handler.changeVisibility(subcategoryNode, subcategoryKey, false);
-        expect(enableSubCategorySpy).to.be.calledWith(subcategoryNode.id, false);
+        await handler.changeVisibility(subCategoryNodes[0], subcategoryKeys[0], false);
+        expect(enableSubCategorySpy).to.be.calledWith(subCategoryNodes[0].id, false);
       });
     });
 
     it("calls enableSubcategoryCategory and enableCategory to ensure that parent category is enabled", async () => {
       await using(createHandler({ activeView: mockViewport().object, categories }), async (handler) => {
         const enableCategorySpy = sinon.stub(handler, "enableCategory");
         const enableSubCategorySpy = sinon.stub(handler, "enableSubCategory");
-        await handler.changeVisibility(subcategoryNode, subcategoryKey, true);
+        await handler.changeVisibility(subCategoryNodes[0], subcategoryKeys[0], true);
         expect(enableCategorySpy).to.be.calledWith(["CategoryId"], true, false);
-        expect(enableSubCategorySpy).to.be.calledWith(subcategoryNode.id, true);
+        expect(enableSubCategorySpy).to.be.calledWith(subCategoryNodes[0].id, true);
         expect(enableCategorySpy.calledBefore(enableSubCategorySpy)).to.be.true;
       });
     });
```

Comment:
something's not right with "enableSubcategoryCategory" in the test name - should be "enableSubCategory"?